api/v1alpha1/circuitbreaker_types.go

```diff
@@ -46,4 +46,20 @@ type CircuitBreaker struct {
 	// +kubebuilder:validation:Maximum=4294967295
 	// +optional
 	MaxRequestsPerConnection *int64 `json:"maxRequestsPerConnection,omitempty"`
+
+	// PerHost defines per-host Circuit Breakers
+	//
+	// +optional
+	PerHost *PerHostCircuitBreakers `json:"perHost,omitempty"`
```

Comment:
should this be called perEndpoint ?

Reply:
I prefer host as it aligns with the upstream field and multiple EG API fields refer to host as well but can change if that's preferred.

Reply:
Is it okay as-is or would you prefer a change here @arkodg?

Reply:
cc @envoyproxy/ai-gateway-maintainers @envoyproxy/gateway-reviewers

Reply:
+1 to endpoint

Reply:
+1 to using endpoint as it aligns better with exiting EG API.

Reply:
Sounds good, updated